docs/ref/checks.txt

```diff
@@ -668,6 +668,7 @@ The following checks are performed when a model contains a
 * **contenttypes.E003**: ``<field>`` is not a ``ForeignKey``.
 * **contenttypes.E004**: ``<field>`` is not a ``ForeignKey`` to
   ``contenttypes.ContentType``.
+* **contenttypes.E005**: ``<model>`` name is too long for :class:`~django.contrib.contenttypes.models.ContentType`'s ``model`` field.
```

Comment:
This should be the same text as the error message: Model names must be at most 100 characters.